### PR TITLE
Releases/2021/4 doc fixes

### PR DESCRIPTION
**Pull request by ntyukaev**

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
16:06:55  ci/prepare-documentation.py:281: trailing whitespace.

File: ci/prepare-documentation.py [hunk 1]
```diff
@@ -269,6 +277,11 @@ def add_model_pages(output_root, parent_element, group, group_title):
 
     sort_titles(group_element)
 
+    title = 'Intel\'s Pre-Trained Models Device Support' if group == 'intel' else 'Public Pre-Trained Models Device Support'
```
Comment (nignatovsky) [inline ci/prepare-documentation.py:280]:
```suggestion
    device_support_title = 'Intel\'s Pre-Trained Models Device Support' if group == 'intel' \
        else 'Public Pre-Trained Models Device Support'
```

File: ci/prepare-documentation.py [hunk 2]
```diff
@@ -269,6 +277,11 @@ def add_model_pages(output_root, parent_element, group, group_title):
...
+    add_page(output_root, group_element,
+             id=f'omz_models_{group}_device_support', path=f'models/{group}/device_support.md',
+             title=title, index=0)
```
Comment (nignatovsky) [inline ci/prepare-documentation.py:283]:
```suggestion
             title=device_support_title, index=0)
```

File: ci/prepare-documentation.py [hunk 3]
```diff
@@ -311,6 +314,16 @@ def main():
     add_model_pages(output_root, trained_models_group_element,
         'public', "Public Pre-trained Models")
 
+    datasets_element = add_page(output_root, navindex_element, id='omz_data_datasets', path='data/datasets.md',
+                            title='Dataset Preparation Guide')
```
Comment (nignatovsky) [inline ci/prepare-documentation.py:318]:
```suggestion
    datasets_element = add_page(output_root, navindex_element, 
        id='omz_data_datasets', path='data/datasets.md', title='Dataset Preparation Guide')
```

File: ci/prepare-documentation.py [hunk 4]
```diff
@@ -311,6 +315,16 @@ def main():
     add_model_pages(output_root, trained_models_group_element,
         'public', "Public Pre-trained Models")
 
+    datasets_element = add_page(output_root, navindex_element,
```
Comment (nignatovsky) [inline ci/prepare-documentation.py:318]:
```suggestion
    datasets_element = add_page(output_root, navindex_element,
```

File: ci/prepare-documentation.py [hunk 5]
```diff
@@ -71,12 +71,20 @@
     'text_to_speech': 'Text-to-speech',
 }
 
-def add_page(output_root, parent, *, id=None, path=None, title=None):
+def add_page(output_root, parent, *, id=None, path=None, title=None, index=-1):
+    if type(index) != int:
+        raise ValueError('index must be a number')
```
Comment (nignatovsky) [inline ci/prepare-documentation.py:76]:
```suggestion
    if not isinstance(index, int):
        raise ValueError('index must be an integer')
```

**Review by vladimir-dudnik — APPROVED** [review]

**Review by nignatovsky — APPROVED** [review]

**Review by andrew-zaytsev — APPROVED** [review]

**Pull request merged**